docs/enhancements/metricsConfig_redesign.md

```diff
@@ -0,0 +1,119 @@
+# HiveConfig.spec.metricsConfig redesign
+
+[HIVE-2344](https://issues.redhat.com/browse/HIVE-2344)
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Current implementations](#current-implementations)
+    - [Duration based threshold](#duration-based-threshold)
+    - [Additional label support](#additional-label-support)
+  - [Goals](#goals)
+  - [Non Goals](#non-goals)
+- [Proposal](#proposal)
+  - [clusterDeploymentRelatedMetrics](#clusterdeploymentrelatedmetrics)
+    - [minimumDuration](#minimumduration)
+    - [additionalClusterDeploymentLabels](#additionalclusterdeploymentlabels)
+    - [clusterDeploymentLabelSelector](#clusterdeploymentlabelselector)
+  - [Implementation Details / Notes](#implementation-details--notes)
+  - [Risks and Mitigations](#risks-and-mitigations)
+
+## Summary
```

Comment:
Let's get a "Reviewers" section. I want to get acks from _at least_ SRE-P (the likes of Naveen/Dustin/Christoph) and OCM (Elad? Priya?) before we proceed with code changes. AppSRE (Patrick/Maor) ACM (maybe start with Gurney, who can delegate) and DPTP (Bear) should also have the opportunity to provide feedback.